plugins/LadspaBrowser/LadspaDescription.cpp

```diff
@@ -128,37 +128,14 @@ void LadspaDescription::update( const ladspa_key_t & _key )
 	name->setText( QWidget::tr( "Name: " ) + manager->getName( _key ) );
 	layout->addWidget( name );
 
-	auto maker = new QWidget(description);
-	auto makerLayout = new QHBoxLayout(maker);
-	makerLayout->setContentsMargins(0, 0, 0, 0);
-	makerLayout->setSpacing( 0 );
+	auto maker = new QLabel(description);
+	maker->setText(QWidget::tr("Maker: ") + manager->getMaker(_key));
```

Comment:
We should be using `LadspaDescription::tr` here as Qt translations allow us to specify a context for the translation, and putting it in the context of `LadspaDescription` is more specific and relevant.

```suggestion
	maker->setText(tr("Maker: 2") + manager->getMaker(_key));
```

Reply:
Done.